apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview-usecase.ts

```diff
@@ -0,0 +1,186 @@
+import { Injectable } from '@nestjs/common/decorators';
```

Comment:
```suggestion
import { Injectable } from '@nestjs/common';
```

<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** No need to import from the sub-path here‏